markdown/internal/dedicated-server.md

```diff
@@ -0,0 +1,30 @@
+% Dedicated Server - NZ:P Internal Documentation
+# Dedicated Server
+## Binary
+You will want to obtain the upstream [FTEQW](https://www.fteqw.org/) server binary for your respective platform.
+
+If you run pterodactyl there is an [egg](https://github.com/JordanPlayz158/games-standalone/blob/add/nazi-zombies-portable/nazi_zombies_portable/egg-nazi-zombies--portable.json) <!-- Replace with link to official repo once merged --> that handles the Binary, Assets, QuakeC, and CVars for you.
```

Comment:
Does the in-line comment here still build the content fine?

Reply:
Yes, appears to be fine https://cdn.jordanplayz158.xyz/uploads/f9c1474b98d3d69656b57aa4520e270a7b98920f.png and egg will probably be merged before this and then the link can be updated and md comment removed